client/coniksclient/internal/cmd/run.go

```diff
@@ -0,0 +1,193 @@
+package cmd
+
+import (
+	"log"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/coniks-sys/coniks-go/client"
+	"github.com/coniks-sys/coniks-go/keyserver/testutil"
+	p "github.com/coniks-sys/coniks-go/protocol"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var help = `
+	- register [name] [key]:
+		Register a new name-to-key binding on the CONIKS-server.
+	- lookup [name]:
+		Lookup the key of some known contact or your own bindings.
+	- exit:
+		Close the REPL and exit the client.
+`
```

Comment:
Somehow the carriage-returns are missing when printing this message. (See https://github.com/coniks-sys/coniks-go/pull/138#issuecomment-264400700) Hence, the help message looks broken.